lib/fluent/plugin/out_copy_ex.rb

```diff
@@ -58,7 +58,11 @@ def emit(tag, es, chain)
       @outputs.each_index do |idx|
         _es = @deep_copy ? es.dup : es
         begin
-          @outputs[idx].emit(tag, _es, NullOutputChain.instance)
+          if @outputs[idx].respond_to?(:emit_events)
```

Comment:
Having `if` statement for each event stream would cause slowness.
Can you create proc objects in `configure` method?